### PR TITLE
Alphanumeric

### PR DESCRIPTION
**Pull request by ekaspern**

### CODE REVIEW

**Comment by darcyadams** [comment]
like to see some tests for this.

**Comment by ekaspern** [comment]
I added tests. Please check again.

File: tests/validation.coffee [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    alphaNumericTest = Validation.isAlphaNumericOnly('string')
+
+    expect(alphaNumericTest).to.equal(true)
+
+  it 'should return the true if numeric', ->
+
+    alphaNumericTest = Validation.isAlphaNumericOnly(14)
+
+    expect(alphaNumericTest).to.equal(true)
```
Comment (darcyadams) [inline tests/validation.coffee]: what should it do if there is a space?

Comment (ekaspern) [inline tests/validation.coffee]: changed the regex to test for space and underscore

**Pull request merged**